### ISSUE TITLE
Multiple dimensions support for `torch.argmax`

### ISSUE THREAD

**Issue by lukas-folle-snkeos** [pytorch/pytorch#105045]
### 🚀 The feature, motivation and pitch

When dealing with multidimensional tensors, getting the location of the maximum values in one function call is very handy. Currently, `torch.argmax` does not support more than one specified dimension as discussed [here](https://discuss.pytorch.org/t/get-indices-of-the-max-of-a-2d-tensor/82150).

A different way of achieving this is described below, but having this functionality in one operation would be nice.

I am open for any input.

### Alternatives

```python
from typing import Sequence
import torch
from torch import Tensor


def argmax(input: Tensor, dims: Sequence) -> Tensor:
    return torch.nonzero(
        torch.Tensor(input == torch.amax(input, dims)[..., *[None]*len(dims)])
    )


if __name__ == "__main__":
    data = torch.rand((1, 66, 64, 64, 64))
    values = argmax(data, (2, 3, 4))  # values.shape (66, 5)
```

**Comment by davidradl** [comment]
@lukas-folle-snkeos I notice that the numpy max function has an axis parameter which can take a tuple of ints. Is this functionally what you are looking for? I do not see any precedent for an `axis` option in PyTorch API.
It seems like the PyTorch `dim` parameter is `axis` without the tuple. If this is what you are looking for - I suggest raising a feature request so this new function could be triaged..

**Comment by lukas-folle-snkeos** [comment]
@davidradl thats exactly what I am looking for. I guess `torch.argmax` would then also be able to support multiple dims if it in turn calls the `torch.max` function.
